content/docs/user-guide/components/reference/terrain/world.md

```diff
@@ -21,9 +29,10 @@ You must add this component to the *Level* entity, the parent of all entities in
 
 | Property | Description | Values | Default |
 | - | - | - | - |
-| World Bounds (min) | The minimum value for the worlds bounds. The **min** value must be less than the **max** value.| Vector3: -2048.0 to 2048.0 | X:`0.0` Y:`0.0` Z:`0.0` |
-| World Bounds (max) | The maximum value for the worlds bounds. The **max** value must be greater than the **min** value. | Vector3: -2048.0 to 2048.0 | X:`1024.0` Y:`1024.0` Z:`1024.0` |
-| Height Query Resolution (m) | The resolution required between each height sample, in meters. | Vector2: 0.001 to Infinity | X:`1.0` Y:`1.0` |
+| Min Height | The minimum value for the height of terrain. The **min** value must be less than the **max** value.| -65536.0 to 65536.0 | 0.0 |
+| Max Height | The maximum value for the worlds bounds. The **max** value must be greater than the **min** value. | -65536.0 to 65536.0 | 1024.0 |
+| Height Query Resolution (m) | The distance between each height sample position, in meters. | 0.1 to Infinity | 1.0 |
+| Surface Data Query Resolution (m) | The distance between each surface data sample position, in meters. | 0.1 to Infinity | 1.0 |
 
 {{< note >}}
 Because large numbers of terrain height samples can take a long time to calculate, O3DE will not allow the number of samples to exceed 8 million.
```

Comment:
Is this note still accurate?

Reply:
No, I think it'll just crash now if you make the values too crazy. That's what we have an open UX ticket about. I'll remove this.